assets/css/main.css

```diff
@@ -0,0 +1,516 @@
+:root {
+  --nw-red: #f72210;
+}
+body {
+  background-color: #f4f5f6;
+  font-family: "Roboto Condensed", sans-serif;
+}
+h5 > a:hover {
+  color: inherit;
+  text-decoration: underline;
+  text-decoration-color: var(--nw-red);
+}
+
+/* BOOTSTRAP OVERRULE */
+
+a {
+  color: inherit;
+}
+.container-fluid {
+  max-width: 1310px;
+  background-color: #fff;
+}
+
+/* HEADER */
+
+header {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  z-index: 10;
+  background-color: var(--nw-red);
+  height: 50px;
+  padding-left: 20px;
+  padding-right: 10px;
+}
+header a:hover {
+  text-decoration: none;
+  color: inherit;
+}
+#meteo {
+  display: none;
+}
+#title {
+  font-size: 2em;
+  font-family: "Rokkitt", serif;
+  line-height: 1.5;
+  font-weight: bold;
+}
+#login {
+  margin-right: 10px;
+}
+#login,
+#subscribe-btn {
+  font-size: 0.8em;
+  font-weight: bold;
+  text-transform: uppercase;
+}
+#subscribe-btn {
+  background-color: #ffe400;
+  border-radius: 99em;
+  padding: 1px 10px;
+  white-space: nowrap;
+}
+#hamburger {
+  font-size: 1em;
+  color: #fff;
+  margin-left: 0.5em;
+}
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `  color: #fff;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```CSS
.white-color{
    color: #fff;
}
```
**Html file**

```HTML
<div class="white-color">
```
**Note:** Do this change to all properties in this file.
- Also for join `color: white;` statement in this same class

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_